lib/services/profiles-api-service.js

```diff
@@ -98,6 +98,17 @@ function profileApiServiceFactory(
                 }
             });
         }
```

Comment:
I think you'll need the same update I made on line 79:

`var query = req.swagger ? req.swagger.query : req.query;`

to get the query properly from both the 1.1 and 2.0 API for this function